zio-json/shared/src/main/scala/zio/json/JsonDecoder.scala

```diff
@@ -129,14 +136,18 @@ trait JsonDecoder[A] extends JsonDecoderPlatformSpecific[A] {
   /**
    * Returns a new codec whose decoded values will be mapped by the specified function.
    */
-  def map[B](f: A => B): JsonDecoder[B] =
+  final def map[B](f: A => B): JsonDecoder[B] =
```

Comment:
It seems this was only `non-final` so that the `option` decoder could override it